docs/DeveloperGuide.md

```diff
@@ -4,29 +4,48 @@ title: Developer Guide
 ---
 * Table of Contents
 {:toc}
+1. [Acknowledgements](#acknowledgements)
+2. [Setting up, getting started](#setting-up-getting-started)
```

Comment:
Maybe can consider changing this to "Setting Up, Getting Started" to standardise on the capitalisation!